### PR TITLE
[solo] Disable unbound using IPv6; we don't need it

### LINKED ISSUE #1116: helios-solo container fails to start
First problem is nmcli which I changed according to #663 issue. After that docker container fails to start with following error:

```
2017/04/25 15:56:08 etcdserver: published {Name:default ClientURLs:[http://localhost:2379 http://localhost:4001]} to cluster 7e27652122e8b2ae
100   387  100   247  100   140    193    109  0:00:01  0:00:01 --:--:--   193
{"action":"set","node":{"key":"/skydns/config","value":"{\"dns_addr\":\"0.0.0.0:5353\", \"ttl\":3600, \"nameservers\": [\"192.168.0.96:53\", \"192.168.180.96:53\", \"192.168.0.69:53\"], \"domain\":\"local.\"}","modifiedIndex":3,"createdIndex":3}}
+ break
+ curl -XPUT http://127.0.0.1:4001/v2/keys/skydns/local/solo -d 'value={"host":"172.17.0.1"}'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   151  100   124  100    27  56724  12351 --:--:-- --:--:-- --:--:-- 62000
{"action":"set","node":{"key":"/skydns/local/solo","value":"{\"host\":\"172.17.0.1\"}","modifiedIndex":4,"createdIndex":4}}
+ unbound
+ skydns
[1493135768] unbound[29:0] error: can't bind socket: Cannot assign requested address
[1493135768] unbound[29:0] debug: failed address ::1 port 8953
[1493135768] unbound[29:0] error: cannot open control interface ::1 8953
[1493135768] unbound[29:0] fatal error: could not open ports
```

### PR DESCRIPTION
**Pull request by davidxia**
A [bug in Docker 17.04][docker-bug] disabled IPv6 causing
helios-solo to crash when trying to run unbound. Unbound tries to
open a control/admin port on localhost 8953 and defaults to listening
on both IPv4 and IPv6.

This change forces unbound to listen **only on IPv4 for all ports**.

Strange things:
  * unbound docs say remote-control->control-enable defaults to no
  * for some reason unbound can listen to port 53 with IPv6

fixes #1116

  [docker-bug]: moby/moby#32433

### CODE REVIEW

**Comment by davidxia** [comment]
cc @caipre @mavenraven

**Comment by mattnworb** [comment]
LGTM

**Comment by davidxia** [comment]
I built and pushed spotify/helios-solo-base:0.8 manually so the next build of spotify/helios-solo will work.

**Comment by davidxia** [comment]
Built and tested helios-solo locally. It's able to start and run.

File: solo/base/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN curl -o $SKYDNS_PLUGIN_DEB -L $SKYDNS_PLUGIN_DEB_URI \
     && rm $SKYDNS_PLUGIN_DEB
 
 # Install Go (from the official golang Dockerfile)
-ENV GOLANG_VERSION 1.6
+ENV GOLANG_VERSION 1.8.1
```
Comment (davidxia) [inline solo/base/Dockerfile]: Needed to be able to build the latest skydns. Didn't want to figure out a way to fix the version of skydns. https://github.com/spotify/helios/issues/901

**Pull request merged**